src/templates/domain/useCases/create.ejs

```diff
@@ -26,7 +26,7 @@ const create<%- props.name.pascalCase %> = injection =>
       ctx.<%- props.name.camelCase %> = <%- props.name.pascalCase %>.fromJSON(ctx.req)
       ctx.<%- props.name.camelCase %>.id = Math.floor(Math.random() * 100000)
       
-      if (!ctx.<%- props.name.camelCase %>.isValid()) 
+      if (!ctx.<%- props.name.camelCase %>.isValid({exceptIDs: true})) 
```

Comment:
Having `{ exceptIDs: true }` in `create` use case should be fine in general, but in this case the `id` is passed on the line above, so the ID should be validated too.

BTW, why `{ exceptIDs: true }` is used on the others use cases?